src/components/material/material-buttons/physical/MaterialButtonsFindOnShelf.tsx

```diff
@@ -0,0 +1,85 @@
+import * as React from "react";
+import { FC } from "react";
+import { useGetAvailabilityV3 } from "../../../../core/fbs/fbs";
+import { useText } from "../../../../core/utils/text";
+import { FaustId } from "../../../../core/utils/types/ids";
+import { Button } from "../../../Buttons/Button";
+
+export interface MaterialButtonsFindOnShelfProps {
+  isButton?: boolean;
+  faustIds: FaustId[];
+}
+
+const MaterialButtonsFindOnShelf: FC<MaterialButtonsFindOnShelfProps> = ({
+  isButton,
+  faustIds
+}) => {
+  const t = useText();
+  const { data, isLoading, isError } = useGetAvailabilityV3({
+    recordid: faustIds
+  });
+
+  const onClick = () => {
+    // we open the find on shelf modal here
+  };
+
+  // If element is currently focused on, we would like to let users open it using enter
+  const onKeyUp = (key: string) => {
+    if (key === "Enter") {
+      onClick();
+    }
+  };
+
+  if (!data || isError || isLoading) {
+    // TODO: handle error here once we handle errors
+    return null;
+  }
+
+  if (isButton) {
+    if (!data[0].available) {
+      return (
+        <Button
+          label={t("materialIsLoanedOutText")}
+          buttonType="none"
+          variant="outline"
+          disabled
+          collapsible={false}
+          size="large"
+        />
+      );
+    }
+    return (
+      <Button
+        label={t("findOnBookshelfText")}
+        buttonType="none"
+        variant="outline"
+        disabled={false}
+        collapsible={false}
+        size="large"
+        onClick={onClick}
+      />
+    );
+  }
+
+  if (!data[0].available) {
+    return (
+      <span className="text-small-caption material-manifestation-item__find capitalize-all">
+        {t("materialIsLoanedOutText")}
+      </span>
+    );
+  }
+
+  return (
+    <span
```

Comment:
why is this not a button?

Reply:
Because that's how it was implemented in the design system. I'm thinking - if we want it to be a button I can make a PR correcting it in the design system and then correct it here too. What do you say @sinejespersen ?

Reply:
Diddly done :)